mLRS/Common/hal/esp/esp-device_conf.h

```diff
@@ -149,6 +149,13 @@
   #define FREQUENCY_BAND_2P4_GHZ
 #endif
 
+#ifdef TX_ELRS_RADIOMASTER_TX16S_2400_ESP32
+  #define DEVICE_NAME "RM TX16S int 2.4G"
```

Comment:
maybe "... Int ..." or "... INT ..."

general question: are all these internal different, and need extra handling by us? Or are there some generics possible?

Reply:
It seems that they don't even use the same pins between two different RM boards!

Reply:
OK.  There seems to be more target sharing possible than I expected which raises the issue of naming again.  What should this be called if it supports all or most of the Radiomaster 2.4 internal modules?